### ISSUE TITLE
Replace Lua basis script with NIL script

### ISSUE THREAD

**Issue by Tricky1975** [TrickyGameTools/Kthura#49]
https://github.com/TrickyGameTools/Kthura/blob/edd9da21ff16b5e7f11826b2d381c18aaeefc758/Assets/Script/BasisScript.lua

Not really, something I need to hurry with as it works properly, anyway

**Comment by Tricky1975** [comment]
Fixed

**Comment by Tricky1975** [comment]
Oh wait... in the commit the wrong issue was closed....

**Comment by Tricky1975** [comment]
This script will no longer be in use once the C++ based editor is done, so this is no longer relevant.
